pkg/datapath/linux/bigtcp/logfields.go

```diff
@@ -1,11 +0,0 @@
-// SPDX-License-Identifier: Apache-2.0
-// Copyright Authors of Cilium
-
-package bigtcp
-
-import (
-	"github.com/cilium/cilium/pkg/logging"
-	"github.com/cilium/cilium/pkg/logging/logfields"
-)
-
-var log = logging.DefaultLogger.WithField(logfields.LogSubsys, "big-tcp")
```

Comment:
With slog the "module" attribute is set instead of "subsys" as that's the granularity at which the logger is created: https://github.com/cilium/hive/blob/main/cell/module.go#L103.